### DISCUSSION TITLE
uBlock Origin and Extension Fingerprins

### DISCUSSION THREAD

**Discussion by Vanerio** [uBlockOrigin/uBlock-issues#2445] [Q&A] [answered]
Hi!
I use two profiles in Brave browser. When using this [this page](https://z0ccc.github.io/extension-fingerprints/) to detect browser extensions fingerprint, in one profile uBlock gets detected, but not in the other one, and I don´t know why.

Two questions:
1. Does anyone know how to avoid it from uBlock conf as a user?
2. Is there anything possible from developer´s side to avoid this fingerprint?

Thank you very much!!

**Comment by gothic-bum** [comment]


**Reply by Vanerio** [reply]
Thank you, gothic-bum, for your interest but that does not solve any of my questions:

In question 1 I ask how to avoid it from **uBlock conf** not by changing my browser.

And you are ignoring what I already said: that in one of my Brave profiles it is not detected (although I can't figure out why).

**Comment by Vanerio** [comment, marked as answer]
I found out why in one of my profiles uBlock was not detected...because I have **uBo development build**...🤦🏻‍♂️

**Reply by uBlock-user** [reply]
https://github.com/gorhill/uBlock/tree/master/src/web_accessible_resources#readme
